packages/cli/src/services/project-initializer.ts

```diff
@@ -42,7 +43,7 @@ export async function initializeGit(config: ProjectInitializerConfig): Promise<v
   try {
     await exec('git init && git add -A && git commit -m "Initial commit"', { cwd: projectDir(config) })
   } catch (e) {
-    throw wrapExecError(e, 'Could not initialize git repository')
+    throw wrapExecError(e, Brand.dangerize('Could not initialize git repository'))
```

Comment:
Maybe we could make the `wrapExecError` function apply the `Brand.dangerize` to any string passed, so we get a consistent way to print script errors, what do you think?

Reply:
Sounds good @javiertoledo to keep it consistent. I have updated `wrapExecError` function to _dangerize_ any prefix passed to it ✨